clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -218,6 +219,30 @@ Expected<StringRef> createOutputFile(const Twine &Prefix, StringRef Extension) {
   return TempFiles.back();
 }
 
+Expected<StringRef> writeOffloadFile(const OffloadFile &File) {
```

Comment:
This function has just moved from below without any changes

Reply:
This is not a function that we added. it comes from the community. We will have to try to improve this upstream if we want to. Thanks for pointer.